chapters/03_feed-forward.tex

```diff
@@ -281,26 +276,25 @@ \section{Conditionals}
         \hline
     \end{tabular}
 
+    We assume the input representation to a feedforward net (after layer-norm) contains three values: a mask $p \in \{0, 1\}$, a first option $\phi$, and second option $\psi$. The goal is to compute the ``ternary'' expression:
+    \begin{equation*}
+        \cif{p}{\phi}{\psi} =
+        \begin{cases}
+            \phi & \textrm{if} \; p \\
+            \psi & \textrm{otherwise.}
+        \end{cases}
+    \end{equation*}
 
-We assume the input representation to a feedforward net (after layer-norm) contains three values: a mask $p \in \{0, 1\}$, a first option $\phi$, and second option $\psi$. The goal is to compute the ``ternary'' expression:
-\begin{equation*}
-    \cif{p}{\phi}{\psi} =
-    \begin{cases}
-        \phi & \textrm{if} \; p \\
-        \psi & \textrm{otherwise.}
-    \end{cases}
-\end{equation*}
-
-We adapt a construction used in \citep{merrill2024the} (Theorem 1, arXiv version) for two-layer ReLU feedforward nets.
-Let $\vec 1$ be a vector of ones.
-In the first layer, we define two neurons:
-\begin{align*}
-    h_1 &= \ReLU(-p \vec 1 + \vec 1 + \phi) \\
-    h_2 &= \ReLU(p \vec 1 - \vec 1 + \psi) .
-\end{align*}
-The second layer then simply computes $h_1 + h_2$, which by construction computes $\cif{p}{\phi}{\psi}$.
+    We adapt a construction used in \citep{merrill2024the} (Theorem 1, arXiv version) for two-layer ReLU feedforward nets.
```

Comment:
`\citep[Theorem 1]{merrill2024the}`

If it's the arXiv version that needs to be cited, then the bib entry should be the arXiv version.